doc/advanced_configuration.rst

```diff
@@ -203,6 +203,23 @@ file. You need to add to the configuration dictionary a key called
         'default_thumb_file'     : 'path/to/thumb/file.png'}}
 
 
+choosing the thumbnail image from multiple figures
+==================================================
+
+For examples that generate multiple figures, the default behavior will use
+the first figure created in each as the thumbnail image displayed in the
+gallery. To change the thumbnail image to a figure generated later in
+an example script, add a comment to the example script to specify the
+number of the figure you would like to use as the thumbnail. For example,
+to use the 2nd figure created as the thumbnail:
+
+.. code-block:: python
+
+    sphinx_gallery_thumbnail_number = 2
+
+The default behavior is ``sphinx_gallery_thumbnail_number = 1``. See
```

Comment:
I would remove "The default behavior is ``sphinx_gallery_thumbnail_number = 1``" since you already mention that the default is to pick the first figure above.